Core/include/Acts/Utilities/BinAdjustmentVolume.hpp

```diff
@@ -48,26 +45,26 @@ BinUtility adjustBinUtility(const BinUtility& bu,
   // Loop over the binning data and adjust the dimensions
   for (auto& bd : bData) {
     // The binning value
-    BinningValue bval = bd.binvalue;
+    AxisDirection bval = bd.binvalue;
```

Comment:
This `binvalue` name stays the same?

Reply:
I decided to not change member variables of `BinUtilitty` and `BinningData` - I do not see the benefit to it when we want to remove this schema relatively soon.